language/de.xml

```diff
@@ -124,6 +126,7 @@
 		<item name="wcf.conversation.noParticipantsWarning"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Du bist{else}Sie sind{/if} dabei, auf eine Konversation ohne weitere Teilnehmer zu antworten. Alle anderen Teilnehmer haben diese Konversation verlassen. Niemand wird {if LANGUAGE_USE_INFORMAL_VARIANT}deine{else}Ihre{/if} Nachricht lesen!]]></item>
 		<item name="wcf.conversation.message.permalink"><![CDATA[Permalink zur {#$startIndex}. Nachricht]]></item>
 		<item name="wcf.conversation.markAsRead.doubleClick"><![CDATA[Konversation durch Doppelklick als gelesen markieren]]></item>
+		<item name="wcf.conversation.error.floodControl"><![CDATA[{if LANGUAGE_USE_INFORMAL_VARIANT}Du hast{else}Sie haben{/if} innerhalb der letzten 24 Stunden bereits {if $limit == 1}eine Konversation{else}{#$limit} Konversationen{/if} gestartet. Bitte {if LANGUAGE_USE_INFORMAL_VARIANT}warte{else}warten Sie{/if} bis zum <strong>{@$notBefore|time}</strong>, bevor {if LANGUAGE_USE_INFORMAL_VARIANT}du{else}Sie{/if} eine neue Konversation {if LANGUAGE_USE_INFORMAL_VARIANT}startest{else}starten{/if}.]]></item>
```

Comment:
I didn't tested this, but what happens when `user.conversation.maxStartedConversationsPer24Hours` is set to zero (or negative values except `-1`)? Then this message is invalid. Maybe a validation for the value to not be zero must be set?

Reply:
It will result in a slightly misleading message that shows an incorrect date of January 1970. The `MIN` will be `NULL`, adding `86400` to `NULL` will result in `86400`.

Reply:
The implementation behaves identically to `1`, the condition to throw the exception is not met if there are no records. The minimum value is `-1`, so any value below `-1` cannot be set through the admin panel.

`0` is technically not a valid, but the current implementation does not permit it to exclude this value in a graceful manner while also supporting `-1`. However, there is already a permission introduced in 5.2 which would allow revoking the start of new conversation already.